components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/saml/SAML2BearerGrantHandler.java

```diff
@@ -412,7 +435,7 @@ public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
             x509Certificate = (X509Certificate) IdentityApplicationManagementUtil
                     .decodeCertificate(identityProvider.getCertificate());
         } catch (CertificateException e) {
-            log.error(e.getMessage(), e);
+            log.error("Error while decoding the certificate.");
             throw new IdentityOAuth2Exception("Error occurred while decoding public certificate of Identity Provider "
                     + identityProvider.getIdentityProviderName() + " for tenant domain " + tenantDomain);
```

Comment:
'e' is not included. is logging necessary here? (if yes, add some context information to the message)